site/docs/get-started/OneClickDemo.md

```diff
@@ -32,12 +34,12 @@ Alternatively you can try Fybrik with Katalog,  a data catalog stub, strictly us
 
 === "Demo with OpenMetaData" 
     ```bash
-    curl https://raw.githubusercontent.com/fybrik/fybrik/master/samples/OneClickDemo/OneClickDemo-OMD.sh | bash -
+    export FYBRIK_VERSION={{ currentRelease|default('master') }}; curl https://raw.githubusercontent.com/fybrik/fybrik/${FYBRIK_VERSION}/samples/OneClickDemo/OneClickDemo-OMD.sh | bash -
```

Comment:
it should be:
curl https://raw.githubusercontent.com/fybrik/fybrik/{{ currentRelease|default('master') }}/samples/OneClickDemo/OneClickDemo-OMD.sh

and with Katalog the same

Reply:
Fixed